### PR TITLE
Fix mobile app

### PR DESCRIPTION
**Pull request by KoalaSat**
## What does this PR do?
Fixes mobile app load. Only robot avatars are still failing.

## Checklist before merging
- [x] Install [pre-commit](https://pre-commit.com/#installation) and initialize it: `pip install pre-commit`, then `pre-commit install`. Pre-commit installs git hooks that automatically check the codebase. If pre-commit fails when you commit your changes, please fix the problems it points out.

### CODE REVIEW

File: frontend/static/federation.json [hunk 1]
```diff
@@ -119,7 +119,7 @@
       "i2p": ""
     },
     "testnet": {
-      "onion": "https://testraliar7xkhos2gipv2k65obykofb4jqzl5l4danfryacifi4t7qd.onion",
+      "onion": "http://testraliar7xkhos2gipv2k65obykofb4jqzl5l4danfryacifi4t7qd.onion",
```
Comment (KoalaSat) [inline frontend/static/federation.json:122]: `https` is not valid and making the request in mobile to fail

Comment (Reckless-Satoshi) [reply frontend/static/federation.json:122]: `https` is likely to fail in browser too, was it working in the browser? Maybe TOR browser is smart enough to replace it for `http` by default. Onion services are usually `http://` as they do not require TLS and it is, in fact, difficult to set up TLS for onion. More info: https://community.torproject.org/onion-services/advanced/https/

**Comment by Reckless-Satoshi** [comment]
Thank you for this fix! Please accept a small tip of 300K Sats for your repeated contributions in the last days.

**Comment by Reckless-Satoshi** [comment]
> lnbc3m1pjk83s3pp54lzpjrz

Missed the expiration for this invoice unfortunately, can you post a new one? Thank you!

**Comment by KoalaSat** [comment]
lnbc3m1pjkwm0lpp54y40y2my20g3z6j47stxrmfmmk05yxr5nftkscghsy9q9wff30zsdqqcqzzsxqyz5vqsp5j0y86g9rn4szkz8r7aagdsfwxlk2fzww2cmyunjjqdf3l98yr6ps9qyyssqpueutjxuvytcqtge7hda3hlv2335gm9awg8r0dj6syjrv4d7htl862pyp0gumyjvgddfsh2287esa8y3mqwslys2jt2gl7nqps8qycsp426te8

**Pull request merged**